### PR TITLE
JCBC-1880 - Update SDC version, fix wait_for_couchbase and fix banner.

### PR DESCRIPTION
**Pull request by mikereiche**
Update SDC version to 4.3.0-M3.
Fix wait_for_couchbase script to ignore indexes in other buckets.
Fix banner to read "Spring Data Travel Sample" (vs Java Travel Sample)

### CODE REVIEW

File: wait-for-couchbase.sh [hunk 1]
```diff
@@ -100,7 +100,7 @@ wait-for $ATTEMPTS \
...
-  '. | del(.indexer) | del(.["travel-sample:def_name_type"]) | map(.items_count > 0) | all' \
+  '. | del(.indexer) |  del(."travel-sample:def_name_type") | with_entries(select(.key|startswith("travel-sample:"))) | map(.items_count != 0) | all' \
```
Comment (maria-robobug) [inline wait-for-couchbase.sh:103]: Ooh, nice one! FYI @osfameron we could benefit from adding this to our other sample apps if it makes waiting for indexes a little faster. 👍

**Review by maria-robobug — APPROVED** [review]
👍 LGTM, just had to rebase and test (all good!).

File: wait-for-couchbase.sh (hunk 1 above)
Comment (osfameron) [reply wait-for-couchbase.sh:103]: Nice! Was that added to speed things up @mikereiche ?
Or in response to a problem? (I'm wondering if you were running with an existing dev database, and your setup caused the script to never complete?)

Comment (mikereiche) [reply wait-for-couchbase.sh:103]: In response to a problem.  If you have another bucket that has an index with items_count == 0, it hangs forever.

Comment (osfameron) [reply wait-for-couchbase.sh:103]: Thanks @mikereiche, so yes, we should definitely add that, good catch!

**Pull request merged**